src/EditorFeatures/TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.cs

```diff
@@ -277,22 +283,22 @@ private protected static CodeActionResolveData CreateCodeActionResolveData(strin
         /// <summary>
         /// Creates an LSP server backed by a workspace instance with a solution containing the markup.
         /// </summary>
-        protected Task<TestLspServer> CreateTestLspServerAsync(string markup)
-            => CreateTestLspServerAsync(new string[] { markup }, LanguageNames.CSharp);
+        protected Task<TestLspServer> CreateTestLspServerAsync(string markup, LSP.ClientCapabilities? clientCapabilities = null)
```

Comment:
client capabilities now must be passed in on server creation instead of on each request (which better matches real behavior anyway)